src/engraving/dom/tiejumppointlist.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+#include "global/types/string.h"
+
+using muse::String;
```

Comment:
`using …` in header files is not really preferred because it may cause conflicts in other cpp files that include this header file. In this case I'd just write `muse::String`.